File: mainwindow/mainwindow.py
#!/usr/bin/python2
# -*- coding: utf-8 -*-

from PyQt4 import QtGui, QtCore

class MainWindow(QtGui.QMainWindow):
	"""
	"""
	
	def __init__(self):
		"""
		"""
		super(QtGui.QMainWindow, self).__init__()
		self.tabWidget_ = QtGui.QTabWidget(self)

	def addPage(self, module, name):
		"""
		
		Arguments:
		- `module`:
		- `name`:
		"""
		self.tabWidget_.addTab(module, name)
		
	def resizeEvent(self, event=None):
		"""
		
		Arguments:
		- `event`:
		"""
		self.tabWidget_.resize(self.size())
		

File: components/curveanalyzer/test/mainwindow.cpp
#include "mainwindow.h"
#include <QtGui/QTabWidget>
#include <QtGui/QResizeEvent>

MainWindow::MainWindow(QWidget *parent)
	: QMainWindow(parent)
{
	tabWidget_ = new QTabWidget(this);
}

void MainWindow::addPage(QWidget *page, const QString &label)
{
	tabWidget_->addTab(page, label);
}

void MainWindow::resizeEvent(QResizeEvent *e)
{
	tabWidget_->resize(e->size());
}



















